readme.md

```diff
@@ -2,7 +2,7 @@ PyRoute for Traveller
 =====================
 
 PyRoute is a trade route generation program for Traveller. PyRoute has been ported to python3, there is a branch with 
-the python2 version.
+the python2 version.  The python3 branch needs Python 3.9+ (or PyPy 7.3).
```

Comment:
```suggestion
the python2 version (no longer maintained).  The released version requires Python 3.9+ (or PyPy 7.3).
```

Reply:
I think that last commit updates the readme as you suggested, @tjoneslo